window/src/os/wayland/frame.rs

```diff
@@ -2,6 +2,8 @@
 //! in smithay_client_toolkit 0.11 which is Copyright (c) 2018 Victor Berger
 //! and provided under the terms of the MIT license.
 
+// TODO: update this for SCTK 0.18 and use this instead of the FallbackFrame
```

Comment:
What do we need to do here now that we're on SCTK 0.19? Does it need to be part of this PR?

Reply:
No, it doesn't need to be, and probably shouldn't be part of this PR. 

Basically it would need to be re-written to implement the DecorationsFrame trait.

Alternatively, just delete this and use the [adwaita implementation](https://github.com/PolyMeilex/sctk-adwaita) instead.